### PR TITLE
Add indexes to subscriptions

### PR DESCRIPTION
**Pull request by steveyken**
This will ensure that searches for unread messages and messages for particular users are indexed.

### CODE REVIEW

**Comment by abhinav098** [comment]
Looks good @steveyken , 👍 
But there are some conflicts in `schema.rb`. Could you please check those?

File: app/controllers/api/v1/orders_controller.rb [hunk 1]
```diff
@@ -26,6 +26,7 @@ class OrdersController < Api::V1::ApiController
           param :beneficiary_id, String
           param :address_id, String
           param :booking_type_id, String, desc: 'Booking type.(Online order or appointment)'
+          param :staff_note, String, desc: 'Notes for internal use'
```
Comment (patrixr) [inline app/controllers/api/v1/orders_controller.rb]: how come we have those changes here ? I added that for the sticky notes (already merged) 🤔

Comment (steveyken) [reply app/controllers/api/v1/orders_controller.rb]: Yeah it's weird. They appeared when I rebased and merged from master. I'll replay my commits on another branch to clean it up.

**Comment by steveyken** [comment]
Opened a cleaner version of this PR. See #653

**Pull request closed without merging**